### ISSUE TITLE
Feature Request: prune-branches option for remote branches that have been merged

### ISSUE THREAD

**Issue by JayAndCatchFire** [Originate/git-town#1025]
I love `prune-branches`, but I often forget to delete branches once they're merged to perennial branches.  How about a command that deletes branches that have already been merged?

**Comment by kevgo** [comment]
Hey @JayAndCatchFire, shouldn't the remote branch get deleted when you `git-town ship` it into a perennial branch?

**Comment by JayAndCatchFire** [comment]
Absolutely.  But not everyone uses git town (or uses it consistently).  I think my request may be for more of a plumbing kind of command, which you mentioned in another ticket isn't the point of git town, so this may be out of scope.  Feel free to close this ticket if that's the case.

**Comment by kevgo** [comment]
Good point, many people merge on Github and then forget to delete the remote branch. I'm not certain we can reliably identify remote branches that can be deleted. 

We could use the Github API to find all the branches that belong to merged PRs and delete those.

**Comment by kemenaran** [comment]
> I'm not certain we can reliably identify remote branches that can be deleted.

A few years ago I hacked a version that looks like this:

```shell
git branch --merged | grep  -v <perennial branches>
```

Never had an issue with this, works quite well :)
